lib/pymedphys/docs/howto/advanced/setup-win.rst

```diff
@@ -36,8 +32,8 @@ More Advanced Options
 * `Setting up OpenSSH on Windows 10`_
 * `Add Jupyter Kernel to Poetry`_
 
-.. _`Setting up OpenSSH on Windows 10`: ../howto/win-open-ssh.html
-.. _`Add Jupyter Kernel to Poetry`: ../howto/add-jupyter-kernel.html
+.. _`Setting up OpenSSH on Windows 10`: /howto/win-open-ssh
+.. _`Add Jupyter Kernel to Poetry`: /howto/add-jupyter-kernel
```

Comment:
is this only relevant on MS Windows/Windows 10, and it's automatic in Linux and Mac?

Reply:
OpenSSH is specific to Windows, (and it's far easier on Linux and Mac). With the Jupyter kernel that's everywhere.